compiler/rustc_typeck/src/astconv/mod.rs

```diff
@@ -1475,7 +1475,7 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
         // Find the type of the associated item, and the trait where the associated
         // item is declared.
         let bound = match (&qself_ty.kind(), qself_res) {
-            (_, Res::SelfTy(Some(_), Some(impl_def_id))) => {
+            (_, Res::SelfTy(Some(_), Some((impl_def_id, _)))) => {
```

Comment:
Wait, I think we have to do the same thing here, don't we?

Will look into this later this week

Reply:
ah no we don't :sweat_smile: 

I missed that this is a different match and github skipped the lines between them